core/shared/src/main/scala/spire/math/Polynomial.scala

```diff
@@ -326,22 +374,8 @@ trait Polynomial[@sp(Double) C] { lhs =>
     Polynomial(terms map f)
 
   /**
-   * Returns this polynomial shifted by `h`. Equivalent to calling
-   * `poly.compose(x + h)`.
-   */
-  def shift(h: C)(implicit ring: Rig[C], eq: Eq[C]): Polynomial[C] =
-    compose(Polynomial.x[C] + Polynomial.constant(h))
-
-  /**
-   * Translates this polynomial by `h`. Equivalent to calling
-   * `poly.compose(x - h)`.
-   */
-  def translate(h: C)(implicit ring: Ring[C], eq: Eq[C]): Polynomial[C] =
```

Comment:
I removed `translate`, since it was basically another name for `shift`. Happy for other opinions. It wasn't being used anywhere though.